lib/internal/Magento/Framework/composer.json

```diff
@@ -20,7 +20,8 @@
         "ext-openssl": "*",
         "lib-libxml": "*",
         "ext-xsl": "*",
-        "symfony/process": "~2.1"
+        "symfony/process": "~2.1",
+        "zendframework/zend-http": "~2.4.6"
```

Comment:
I have taken the requirement from `magento/magento2ce` [composer.json](https://github.com/magento/magento2/blob/2.0.4/composer.json#L39) and that is `~2.4.6` at the moment.